src/layouts/guides.html

```diff
@@ -161,6 +183,9 @@
 
     <div class="content-outlet">
       <content-outlet></content-outlet>
+      <div class="content-footer">
+        <app-edit-on-github route="${globalThis.page.route}"></app-edit-on-github>
```

Comment:
Would it be possible we keep it on the bottom right corner for mobile and desktop?  Also, maybe we need to add a bottom margin or something since it seems to be overlapping the footer a bit.

![Screenshot 2024-10-14 at 11 57 04 AM](https://github.com/user-attachments/assets/a8911a0b-1474-444f-ba08-bcad09eab5f7)

Reply:
I double checked Chrome, FF, and Safari and all seem to have a decent margin now. Let me know if you see differently, and which break point you're at.

Reply:
Yeah, maybe it was a zoom in thing or something, but looks fine now.  👍